fastlane/lanes/screenshots.rb

```diff
@@ -155,28 +154,11 @@
   end
 
 
-  #####################################################################################
-  # trigger_screenshots
-  # -----------------------------------------------------------------------------------
-  # This lane triggers a CircleCI workflow on demand to generate Raw and Promo screenshots
-  # -----------------------------------------------------------------------------------
-  # Usage:
-  #   bundle exec fastlane trigger_screenshots [token:<circleci_token>] [fork:<github_fork_name>] [branch:<git_branch>
-  # Notes:
-  #   - The CircleCI token can be passed via the CIRCLECI_TOKEN env var instead of as a parameter to the lane call
-  #   - The fork parameter allows us to trigger a build on a fork of WPAndroid instead of the default repo on `wordpress-mobile` org.
-  #####################################################################################
-  lane :trigger_screenshots do |options|
-    new_options = options.merge({ parameters: { "generate_screenshots": true } })
-    trigger_ci(new_options)
```

Comment:
For the record, I had a look for `trigger_ci` references and saw there weren't any.